RenderStateNotation/CMakeLists.txt

```diff
@@ -26,24 +26,40 @@ file(COPY ../.clang-format DESTINATION "${RSN_PARSER_GENERATED_HEADERS_DIR}")
 
 find_package(Python3 REQUIRED)
 
-set(LIBCLANG_INSTALL_CMD ${Python3_EXECUTABLE} -m pip install libclang==16.0.6)
-set(JINJA2_INSTALL_CMD ${Python3_EXECUTABLE} -m pip install jinja2)
-
-if(${Python3_VERSION} VERSION_GREATER_EQUAL "3.12")
-    set(LIBCLANG_INSTALL_CMD ${LIBCLANG_INSTALL_CMD} --break-system-packages)
-    set(JINJA2_INSTALL_CMD ${JINJA2_INSTALL_CMD} --break-system-packages)
-endif()
+# Ensure libclang is installed
+execute_process(
+    COMMAND ${Python3_EXECUTABLE} -c "import clang"
+    RESULT_VARIABLE LIBCLANG_EXISTS
+)
 
-execute_process(COMMAND ${LIBCLANG_INSTALL_CMD}
+if(NOT LIBCLANG_EXISTS EQUAL "0")
```

Comment:
This condition may be too liberal as there are multiple other options for false value in CMake.
https://cmake.org/cmake/help/latest/command/if.html#constant
It should just be `if(NOT LIBCLANG_EXISTS)`